src/navigation/Navigator.tsx

```diff
@@ -74,36 +86,51 @@ const Navigator = (): JSX.Element => {
       <Stack.Navigator initialRouteName='Profession' screenOptions={TransitionPresets.SlideFromRightIOS}>
         <Stack.Screen options={{ headerShown: false }} name='Profession' component={ProfessionScreen} />
         <Stack.Screen
-          options={({ navigation }) =>
-            defaultOptions(labels.general.header.overview, BackButton, navigation, 'Profession')
+          options={({ route, navigation }) =>
+            defaultOptions(
+              route.params.extraParams.parentTitle
+                ? route.params.extraParams.parentTitle
+                : labels.general.header.overview,
+              BackButton,
+              navigation,
+              !!route.params.extraParams.parentTitle
+            )
           }
           name='ProfessionSubcategory'
           component={ProfessionSubcategoryScreen}
         />
         <Stack.Screen
           options={({ route, navigation }: any) =>
-            defaultOptions(route.params.extraParams.disciplineTitle, BackButton, navigation, 'ProfessionSubcategory')
+            defaultOptions(route.params.extraParams.disciplineTitle, BackButton, navigation, true)
```

Comment:
Why did you remove the `ProfessionSubcategory` string? Not needed anymore?

Reply:
Yes, not needed.